cwltool/workflow_job.py

```diff
@@ -88,12 +88,16 @@ def __init__(
     ) -> None:
         """Initialize."""
         self.dest = dest
-        self.completed = 0
+        self._completed = set()
```

Comment:
```suggestion
        self._completed: Set[int] = set()
```
I think that is the correct type